eng/pipelines/runtime-official.yml

```diff
@@ -90,6 +93,9 @@ stages:
       jobTemplate: /eng/pipelines/libraries/build-job.yml
       buildConfig: Release
       platforms:
+      - iOS_x64
```

Comment:
Consider splitting these as well as they don't honor the comment for this block.